examples/chip-tool/templates/commands.zapt

```diff
@@ -36,6 +36,30 @@ CHIP_ERROR LogValue(const char * label, size_t indent, {{zapTypeToDecodableClust
 {{/zcl_structs}}
 {{/zcl_clusters}}
 
+void LogEventHeader(const chip::app::EventHeader & eventHeader)
```

Comment:
This function does not seem to depend on  parameters  given to the template. Please move to a propper C++/header as there is no reason for this to be "codegen".